res.inp

```diff
@@ -7,4 +7,4 @@ State:              Maryland
 Zip Code:           20902
 
 [Search Control]
-Search Box Half-Width (miles): 0.5
\ No newline at end of file
+Search Box Half-Width (miles): 0.1

```

Comment:
Ok good job @AmritPatel, we'll keep this at 0.1 during dev stages so it runs quicker.